public/app/features/integrated-alerting/IntegratedAlertingPage.tsx

```diff
@@ -47,6 +50,62 @@ const IntegratedAlertingPage: FC = () => {
         ))}
       </TabsBar>
       <TabContent>{tabs.find(tab => tab.key === activeTab).component}</TabContent>
+    </>
+  );
+};
+
+export const IntegratedAlertingContent: FC<{ loadingSettings: boolean; alertingEnabled: boolean }> = ({
```

Comment:
We should keep one component per file, this three components can be separated. Also this one should have an interface for the props.

Reply:
Done 👍